docs/blog/content/material/database.en.md

```diff
@@ -4,4 +4,296 @@ weight = 6
 chapter = true
 +++
 
-## TODO
+## How we build a distributed database
+
+Author | Liang Zhang
+
+  In the past few decades, relational database continuously occupy an absolute dominant position in the field of databases. The advantages of relation database, like stability, safety, ease of use that are the cornerstone of a modern system. With the rapid development of the Internet, databases built on stand-alone systems are no longer able to meet higher and higher concurrent requests and larger and larger data storage requirements. Therefore, distributed databases are more and more widely adopted.
```

Comment:
The advantages of relational database, like stability, safety, ease of use that are the cornerstone of a modern system.